.github/workflows/_integration_tests.yml

```diff
@@ -153,6 +153,7 @@ jobs:
           # Wait a few seconds for the services to fully start. GH runners are
           # slow, so this gives the Client enough time to initialize its tun interface,
           # for example.
+          # Intended to mitigate <https://github.com/firezone/firezone/issues/5830>
```

Comment:
Clarify that Jamil's change here will also apply to IP Resources, and it is still needed for Docker-based Clients until we figure out how to do signal service health for Docker.

This PR, #6026, only applies to tests where the Client starts from `systemd`, and I think that's only for testing the `systemd-resolved` DNS control method.